### DISCUSSION TITLE
Wrong source translation interpretation

### DISCUSSION THREAD

**Discussion by ilxlodev** [WeblateOrg/weblate#11324] [Q&A] [answered]
Hello!

I'm pretty new to Weblate and my problem may be pretty stupid.

I come from Crowdin and I have a translation structure similar to this:

```json
{
  "SOMETHING_WENT_WRONG": "Something has gone wrong on our side!"
}
```

_Basically, a key and translation pair._

The thing is, I have set the file format to i18next JSON v4, as that seems to be the best fit, however, Weblate shows the "key" of the translation as the context to translate, which (looking at the example) is quite complicated to guess.

I do not rule out having a problem with the configuration, therefore, I leave the configuration used here:
- **File format**: `i18next JSON file v4`.
- **File mask**: `*/errors.json`.
- **Language filter**: _Default_.
- **Source language**: `English (United States)`.
- **Monolingual base language file**: _Empty_.
- **Edit base file**: _Enabled_.
- **Intermediate language file**: _Empty_.
- **Template for new translations**: `en-US/errors.json`.
- **Adding new translation**: _Default_.
- **Language code style**: _Default_.
- **Screenshot file mask**: _Empty_.

Thanks for your time, if you need more context I'll add it as soon as I can.

**Comment by nijel** [comment, marked as answer]
You need `Monolingual base language file`, see https://docs.weblate.org/en/latest/formats.html#bilingual-and-monolingual-formats

**Reply by ilxlodev** [reply]
Thank you so much!! <3
